features/dataset-metadata-citation-doi.feature

```diff
@@ -0,0 +1,14 @@
+@issue-85 @dataset-metadata-citation-doi
+Feature: Add the metadata schema on dataset page to allow Hypothesis to parse citation DOI
+  As a researcher
+  I want the Hypothesis annotation tool to recognise a GigaDB dataset page url as an alias of its DOI
+  So that I can correctly cite it irrespective of which website it appears on or I annotate from
+
+  @wip @javascript @issue-85
+  Scenario: Go to dataset page and there is a meta-tag with DOI
+    Given I am not logged in to Gigadb web site
+    When I go to "/dataset/100002"
+    Then I should see "Genomic data from Adelie penguin (Pygoscelis adeliae)"
+    And There should be a meta-tag element has attribute "name" with value "citation_doi"
+    And There should be a meta-tag element has attribute "content" with value "10.5072/100002"
```

Comment:
Hi @rija This acceptance test has to use doi `10.5072/10002` to pass, the doi `10.5524/100002` you mentioned is not working.  Is `10.5072/10002` the correct doi? :rofl:

Reply:
@kencho51 
yes, ``10.5072`` is fine.